config_files/db_fields.yaml

```diff
@@ -0,0 +1,273 @@
+file_name: "db_filename"
```

Comment:
These kinds of files should be moved to a different directory -- we should think about where these descriptive/metadata files should go -- keeping them separate from "configuration" files (which should dictate what the running program needs to do)